src/main/java/org/codeit/roomunion/meeting/adapter/out/persistence/MeetingRepositoryImpl.java

```diff
@@ -0,0 +1,60 @@
+package org.codeit.roomunion.meeting.adapter.out.persistence;
+
+import lombok.RequiredArgsConstructor;
+import org.codeit.roomunion.common.exception.CustomException;
+import org.codeit.roomunion.common.exception.UserNotFoundException;
+import org.codeit.roomunion.meeting.adapter.out.persistence.entity.MeetingEntity;
+import org.codeit.roomunion.meeting.adapter.out.persistence.jpa.MeetingJpaRepository;
+import org.codeit.roomunion.meeting.adapter.out.persistence.jpa.MeetingMemberJpaRepository;
+import org.codeit.roomunion.meeting.application.port.out.MeetingRepository;
+import org.codeit.roomunion.meeting.domain.model.Meeting;
+import org.codeit.roomunion.meeting.domain.model.command.MeetingCreateCommand;
+import org.codeit.roomunion.meeting.domain.model.enums.MeetingRole;
+import org.codeit.roomunion.meeting.exception.MeetingErrorCode;
+import org.codeit.roomunion.user.adapter.out.persistence.entity.UserEntity;
+import org.codeit.roomunion.user.adapter.out.persistence.jpa.UserJpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class MeetingRepositoryImpl implements MeetingRepository {
+
+    private final MeetingJpaRepository meetingJpaRepository;
+    private final MeetingMemberJpaRepository meetingMemberJpaRepository;
+    private final UserJpaRepository userJpaRepository;
+
+    @Override
+    public Meeting createMeeting(MeetingCreateCommand command) {
+        // TODO 현태님 UserErrorCode로 변경하시면 예외처리 적용
+        UserEntity hostUser = userJpaRepository.findById(command.getUserId())
+            .orElseThrow(UserNotFoundException::new);
+
+        MeetingEntity meetingEntity = MeetingEntity.from(command, hostUser);
+
+        MeetingEntity savedMeetingEntity = meetingJpaRepository.save(meetingEntity);
+
+        return savedMeetingEntity.toDomain(hostUser.getId(), hostUser.getNickname());
+    }
+
+
+    @Override
+    public Meeting findById(Long meetingId) {
+        return meetingJpaRepository.findById(meetingId)
```

Comment:
p2; 사실 이 코드를 읽고 해석하는데 너무 오래 걸렸어요... 너무 복잡해요.
stream api를 사용해서 처리하는건 좋은데 한번에 너무 많은 일들을 하나의 메서드로 처리하려고 하셔서 복잡한거같아요. 아래처럼 하는건 어떨까요?
MeetingEntity와 MeetingMemberEntity가 없는 상태는 무조건 문제가 있는 상황이라 바로 예외 던지면 될것같습니다. 이렇게 처리하시게 된다면 empty는 필요없겠네유

```suggestion
MeetingEntity meetingEntity = meetingJpaRepository.findById(meetingId)
            .orElseThrow(Exception::new);

        MeetingMemberEntity memberEntity = meetingMemberJpaRepository.findBymeetingIdAndMeetingRole(meetingId, MeetingRole.HOST)
            .orElseThrow(Exception::new);

        return meetingEntity.toDomain(memberEntity);
```